tsbridge/config.go

```diff
@@ -157,17 +154,15 @@ func NewConfig(ctx context.Context, opts *ConfigOptions) (*Config, error) {
 		}
 	}
 
-	log.Debugf(ctx, "Read %d metrics and %d destinations from the config file", len(metrics), len(destinations))
+	log.WithContext(ctx).Debugf("Read %d metrics and %d destinations from the config file", len(metrics), len(destinations))
 	return c, nil
 }
 
-// projectID returns the name of the App Engine app that code is running in.
-// Empty string is returned if code is running in dev_appserver.py
-func projectID(ctx context.Context) string {
-	id := appIDFunc(ctx)
-	// dev_appserver.py returns "None" as a string. ¯\_(ツ)_/¯
-	if id == "None" {
+// projectID returns the name of the GCP project that code is running in.
+func projectID() string {
+	value, exists := os.LookupEnv("GOOGLE_CLOUD_PROJECT")
```

Comment:
Do this in a single line:

```go
return os.Getenv("GOOGLE_CLOUD_PROJECT")
```

Reply:
This needs to return empty if var doesn't exist due to legacy logic in the NewConfig, I will make an effort to refactor it later though.